src/specs/api.yaml

```diff
@@ -630,6 +630,18 @@ paths:
             application/json:
               schema:
                 $ref: ./models/HTTPError.v1.yaml
+        '401':
+          description: The request lacks authentication credentials.
+          content:
+            application/json:
+              schema:
+                $ref: ./models/HTTPError.v1.yaml
+        '403':
+          description: The request lacks authentication credentials.
```

Comment:
I think that in this case the user is authenticated, but lacks authorization

Reply:
Thanks for the catch! Changed to better reflect it.